src/ints/mouse.cpp

```diff
@@ -888,100 +915,163 @@ static Bitu INT33_Handler(void) {
+        //       CX = 0 for first, != 0 for next
+        //       Exit:
+        //       BX:DX = named string far ptr
+        //       CX = video mode number
+        LOG(LOG_MOUSE,LOG_ERROR)("Enumerate video modes not implemented");
+        break;
+	case 0x2a: // MS MOUSE v7.01+ - get cursor hot spot
 		reg_al=(uint8_t)-mouse.hidden;	// Microsoft uses a negative byte counter for cursor visibility
 		reg_bx=(uint16_t)mouse.hotx;
 		reg_cx=(uint16_t)mouse.hoty;
-		reg_dx=0x04;	// PS/2 mouse type
+		reg_dx=0x04; // PS/2 mouse type
 		break;
-	case 0x31: /* Get Current Minimum/Maximum virtual coordinates */
+    case 0x2b: // MS MOUSE v7.0+ - load acceleration profiles
+        LOG(LOG_MOUSE,LOG_ERROR)("Load acceleration profiles not implemented");
+        break;
+    case 0x2c: // MS MOUSE v7.0+ - get acceleration profiles
+        LOG(LOG_MOUSE,LOG_ERROR)("Get acceleration profiles not implemented");
+        break;
+    case 0x2d: // MS MOUSE v7.0+ - select acceleration profile
+        LOG(LOG_MOUSE,LOG_ERROR)("Select acceleration profile not implemented");
+        break;
+    case 0x2e: // MS MOUSE v8.10+ - set acceleration profile names
+        LOG(LOG_MOUSE,LOG_ERROR)("Set acceleration profile names not implemented");
+        break;
+    case 0x2f: // MS MOUSE v7.02+ - mouse hardware reset
+        LOG(LOG_MOUSE,LOG_ERROR)("INT 33 AX=2F mouse hardware reset not implemented");
+        break;
+    case 0x30: // MS MOUSE v7.04+ - get/set BallPoint information
+        LOG(LOG_MOUSE,LOG_ERROR)("Get/set BallPoint information not implemented");
+        break;
+	case 0x31: // MS MOUSE v7.05+ - get current minimum/maximum virtual coordinates
 		reg_ax=(uint16_t)mouse.min_x;
 		reg_bx=(uint16_t)mouse.min_y;
 		reg_cx=(uint16_t)mouse.max_x;
 		reg_dx=(uint16_t)mouse.max_y;
 		break;
+    case 0x32: // MS MOUSE v7.05+ - get active advanced functions
+        LOG(LOG_MOUSE,LOG_ERROR)("Get active advanced functions not implemented");
+        break;
+    case 0x33: // MS MOUSE v7.05+ - get switch settings and accelleration profile data
+        LOG(LOG_MOUSE,LOG_ERROR)("Get switch settings and acceleration profile data not implemented");
+        break;
+    case 0x34: // MS MOUSE v8.0+ - get initialization file
+        LOG(LOG_MOUSE,LOG_ERROR)("Get initialization file not implemented");
+        break;
+    case 0x35: // MS MOUSE v8.10+ - LCD screen large pointer support
+        LOG(LOG_MOUSE,LOG_ERROR)("LCD screen large pointer support not implemented");
+        break;
+    case 0x4d: // MS MOUSE - return pointer to copyright string
+        LOG(LOG_MOUSE,LOG_ERROR)("Return pointer to copyright string not implemented");
+        break;
+    case 0x6d: // MS MOUSE - get version string
+        LOG(LOG_MOUSE,LOG_ERROR)("Get version string not implemented");
+        break;
+    case 0x53C1: // Logitech CyberMan
+        LOG(LOG_MOUSE,LOG_NORMAL)("Mouse function 53C1 for Logitech CyberMan called. Ignored by regular mouse driver.");
```

Comment:
Some minor formatting issues for this block of code (no space after comma, line length issues, etc.) Just run it thorugh the clang formatter.

Reply:
Used the script to correct all the formatting.